### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by AnelisaDee**
**Project Description**

- Used media queries to make sure that your website displays correctly on desktop screens.
- In order to lay out the elements on the page, I used Grid and Flexbox.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -46,36 +66,37 @@ <h2>I’am a Software Developer</h2>
...
+        <ul class="work-list">
           <a href="#"><li class="item work-item-1">css</li></a>
           <a href="#"><li class="item work-item-2">html</li></a>
           <a href="#"><li class="item work-item-3">bootstrap</li></a>
           <a href="#"><li class="item work-item-4">Ruby</li></a>
         </ul>
-      </div>
       <div class="link">
         <a href="#">See Project</a>
       </div>
```
Comment (omarsalem7) [inline index.html]: - [ ] Kindly consider adding appropriate padding and margin to match your design like the one in the Figma

| Your design  | Figma design|
| ------------- | ------------- |
| image | image|

File: index.html [hunk 2]
```diff
@@ -119,11 +146,14 @@ <h1>Profesional Art Printing Data</h1>
...
+   </div>
       <div class="project-list">
         <a href="#">See Project</a>
       </div>
+       
       </div>
-      <div class="project">
+      <div class="four-project">
+        <div class="project-content">
```
Comment (omarsalem7) [inline index.html]: - [ ] Also consider making project cards like the one in the template in height and width and also fix height for it in the mobile version and also make sure overlay content.

| Your design  | Figma design|
| ------------- | ------------- |
| image  | image |

File: index.html [hunk 3]
```diff
@@ -221,13 +268,16 @@ <h2 class="contact-heading">
...
+        <footer>
         <div class="contact-icon">
           <img src="./Pictures/SocialMedia2.png" alt="Social media">
         </div>
         <div class="shape">
           <img src="./Pictures/Shape.png" alt="shape">
         </div>
-    </div>
+    </footer>
 </body>
```
Comment (omarsalem7) [inline index.html:303]: - [ ] Kindly consider fixing the design of the contact to match the Figma template.

| Your design  | Figma design|
| ------------- | ------------- |
| image  | image |

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @AnelisaDee 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title and description. ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by fobadara — APPROVED** [review]
Hi @AnelisaDee, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Linters are passing

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sajid-munawar — APPROVED** [review]
### Hi @AnelisaDee , 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
 ## [Optional] suggestions:
 * [ ]  Please check my inline comments.

> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**